.changes/v3.4.0/704-notes.md

```diff
@@ -0,0 +1 @@
+* Drop support for EOL VCD 10.0 [GH-704]
```

Comment:
Please also add a date when 10.0 goes.

Reply:
For previous versions, our communication was always "Drop support for VCD X.X" without mention of EOL.

Reply:
I vote for just keeping it plain simple without EOL mentions. Changed.